pom.xml

```diff
@@ -114,11 +114,13 @@
                 <groupId>com.google.code.findbugs</groupId>
                 <artifactId>jsr305</artifactId>
                 <version>3.0.1</version>
+                <scope>provided</scope>
             </dependency>
             <dependency>
                 <groupId>com.google.gag</groupId>
                 <artifactId>gag</artifactId>
                 <version>1.0.1</version>
+                <scope>provided</scope>
```

Comment:
There are only annotations in there and the java runtime allows annotations to be absent, i.e. not on the classpath.